Review extension

jpmckinney opened the pull request:
sustainabilityGoal.csv

* [x] Where does the term "formal employment" come from?
* [x] Should innovation be under economic?

sustainabilityStrategy.csv

* [x] "for example, demanding SPP-related tasks as part of the contract's implementation" is not a specific example. Can we replace it with one?

jpmckinney committed:
extension.json: Update to GitHub description

jpmckinney committed:
readme: Fix test failures

jpmckinney committed:
codelists: smesInclusion -> smeInclusion, womanInclusion -> womenInclusion. Remove repetition of base code's title in sub-code titles.

jpmckinney committed:
readme: Fix test failures

yolile commented:
> Where does the term "formal employment" come from?

From the toolkit https://sustainable.open-contracting.org/what-is-spp-and-open-spp (see the image)

jpmckinney committed:
codelists: Copy-edit goal descriptions

yolile commented:
> Should innovation be under economic?

I understood we agreed to keeping it separate as per https://github.com/open-contracting/standard/issues/1543#issuecomment-1218525523

jpmckinney committed:
codelists: Use sentence case for titles. Copy-edit descriptions and change examples

jpmckinney commented:
Hmm, well I had written:

> However, I think we should try to stick to the sustainability/social objectives/innovation framing rather than forcing a mapping to environmental/social/ economic, because the relationship I described above might not always hold

But since the SPP guide uses environmental/social/economic, we should follow that, in which innovation is part of economic. Otherwise, we end up mixing the two different approaches to conceptual hierarchy.

jpmckinney committed:
codelists: Fix quoting from 5bdffa3d4a523ec3e46440694d757b596d9e19dd

jpmckinney committed:
release-schema: Follow style guide

- Do not "Declare the type of the field: for example, “A list”, “A true/false field”, etc."
- Do not list the fields within an object as part of its description.

https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#field-and-code-descriptions

yolile commented:
> "for example, demanding SPP-related tasks as part of the contract's implementation" is not a specific example. Can we replace it with one?

From the [toolkit](https://sustainable.open-contracting.org/implement/prepare-contract-obligations), one example could be "For example, requiring suppliers continuous monitoring of sustainability-related indicators, such as carbon emissions during the contract. "

jpmckinney committed:
release-schema: Style guide: "Field names should not include their parent\’s name"

jpmckinney committed:
release-schema: Copy-edit sustainability fields. Align code descriptions. Update example for contractPerformanceConditions.

jpmckinney committed:
release-schema: Can't include parent/ancestor in single-string field

jpmckinney committed:
readme: Codelist is open, not closed. Defer details to schema and codelists. Move external references to Background section. Copy-edit.

jpmckinney commented:
All done - just need to decide on 'innovation'

yolile commented:
> All done - just need to decide on 'innovation'

I guess the options are, (1) remove it from the codelist completely, (2) add economic.marketInnovationPromotion, (3) add _social.innovationSomething,_ or (2) and (3)

jpmckinney commented:
Let's do (2) for now and see if there is evidence for (3) over time.

yolile committed:
sustainabilityGoal codelist: move innovation under economic

yolile commented:
Done

jpmckinney commented on codelists/sustainabilityGoal.csv:
```diff
@@ -1,15 +1,15 @@
+economic,Economic,"The goal is economic, such as formal employment and the local economy."
+economic.formalEmploymentPromotion,Formal employment promotion,The goal is to promote formal employment.
+economic.localEconomyPromotion,Local economy promotion,The goal is to promote the local economy.
+economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote the market innovation.
```
```suggestion
economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote market innovation.
```

jpmckinney reviewed (commented):
Looks good, just one edit before merge.

yolile committed:
Update codelists/sustainabilityGoal.csv

Co-authored-by: James McKinney <26463+jpmckinney@users.noreply.github.com>

yolile committed:
sustainabilityStrategy codelist: add reservedParticipation

yolile commented:
@jpmckinney I've added `reservedParticipation` in https://github.com/open-contracting-extensions/ocds_sustainability_extension/pull/2/commits/8a125bdadcd934841a45272ff83db3d46a54281c

jpmckinney committed:
codelists: Make contracting process singular

jpmckinney commented:
Ready to merge – just fixed the pluralization.
